Marlin/src/lcd/language/language_sk.h

```diff
@@ -259,7 +264,9 @@
 #define MSG_STORE_EEPROM                    _UxGT("Uložiť nastavenie")
 #define MSG_LOAD_EEPROM                     _UxGT("Načítať nastavenie")
 #define MSG_RESTORE_FAILSAFE                _UxGT("Obnoviť nastavenie")
-#define MSG_INIT_EEPROM                     _UxGT("Inicializácia EEPROM")
+#define MSG_INIT_EEPROM                     _UxGT("Inicializovať EEPROM")
+#define MSG_SD_UPDATE                       _UxGT("Aktualizovať z SD")
+#define MSG_RESET_PRINTER                   _UxGT("Reštartovať tlačiar.")
```

Comment:
These are just a tiny bit too long. Menu titles over 19 characters won't fit on a character display. (And sub-menu titles must be 18 characters or fewer.)
```
1234567890123456789|0
Inicializovať EEPRO|M
Reštartovať tlačiar|.
```

Reply:
Thank you @thinkyhead. I will correct the strings, I thought limit was 20 characters.